src/components/Book.js

```diff
@@ -1,25 +1,60 @@
 import PropTypes from 'prop-types';
+import { CircularProgressbar, buildStyles } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
 import { useDispatch } from 'react-redux';
-import { removeBook } from '../redux/books/booksSlice';
+import { deleteBook } from '../redux/books/booksSlice';
 
-const Book = ({ title, author, id }) => {
+const Book = ({
+  title, author, Id, category,
+}) => {
   const dispatch = useDispatch();
 
   return (
-
     <div className="book">
-      <p className="title">{title}</p>
-      <p>{author}</p>
-      <button type="button" onClick={() => { dispatch(removeBook(id)); }}>Remove</button>
+      <div className="bookinfo">
+        <p className="category">{category}</p>
+        <p className="title">{title}</p>
+        <p className="author">{author}</p>
+        <div className="buttons">
+          <button className="button comment" type="button" onClick={() => { dispatch(deleteBook(Id)); }}>Comments</button>
+          |
+          <button className="button" type="button" onClick={() => { dispatch(deleteBook(Id)); }}>Remove</button>
+          |
+          <button className="button" type="button" onClick={() => { dispatch(deleteBook(Id)); }}>Edit</button>
+        </div>
+      </div>
+      <div className="progress">
+        <CircularProgressbar
+          value={45}
+          text={`${45}%`}
+          styles={buildStyles({
+            rotation: 0.25,
+
+            // Whether to use rounded or flat corners on the ends - can use 'butt' or 'round'
+            strokeLinecap: 'round',
+            textSize: '18px',
+            pathTransitionDuration: 0.5,
+            pathColor: `rgba(62, 152, 199, ${45 / 100})`,
+            textColor: '#f88',
+            trailColor: '#d6d6d6',
+            backgroundColor: '#4386bf',
+          })}
+        />
+      </div>
+      <div className="update">
+        <p className="Current-Chapter">CURRENT CHAPTER</p>
+        <p className="Current-Lesson">Chapter 17</p>
+        <button type="button" className="Rectangle-2">UPDATE PROGRESS</button>
+      </div>
     </div>
   );
 };
```

Comment:
- [ ] Great job so far. However, there's few things you need to work on before you move on to the next milestone. It is required that your design must look exactly like the zeplin design. This includes every element on the zeplin design. At this moment, there are some differences between your design and zeplin design. Take a look :point_down: 

-  There's one image icon to the right of the zeplin design which is absent on your design. Kindly fix that.
### Your design
![your-design-1](https://user-images.githubusercontent.com/26597432/232202965-c1e4f56e-7581-4686-9dea-0f675fddf007.png)

### Zeplin design
![zeplin-1](https://user-images.githubusercontent.com/26597432/232202984-6a4f1632-0ace-4acf-86bd-28a24f5aadc4.png)


-  The circular design which meant to read the percentage on your design is different to what we have on the zeplin design. Kindly fix that too.

### Your design
![your-design-1](https://user-images.githubusercontent.com/26597432/232202884-cf463c44-c2f4-4e09-b02f-d79a45ad0a19.png)

### Zeplin design
![zeplin-1](https://user-images.githubusercontent.com/26597432/232202896-30e61031-8e04-40bd-aee9-bc0d783f43d4.png)


- The main title (BOOKSTORE CMS) on your design is not properly vertically aligned with the rest of the elements on the page as oppose to how it is on the zeplin. Kindly look into that and fix.
### Your design
![your-main-title-align](https://user-images.githubusercontent.com/26597432/232202860-de2c3d08-a315-4e30-8452-baedc48b3ae0.png)

### Zeplin design
![zeplin-main-title-align](https://user-images.githubusercontent.com/26597432/232202864-383a8fdd-c076-402c-bea1-38cfd68e8bbb.png)


### Your Design
![your-design-1](https://user-images.githubusercontent.com/26597432/232202486-f34a4195-b60c-483e-be93-1bc161fe05f3.png)

### Zeplin Design
![zeplin-1](https://user-images.githubusercontent.com/26597432/232202492-ccbc53e1-01eb-4f99-9751-38a63487b49e.png)